### DISCUSSION TITLE
why "bot accounts" can not see imported sessions? - Solved

### DISCUSSION THREAD

**Discussion by omidRMR** [masterking32/MasterCryptoFarmBot#23] [Q&A] [answered]
Why "bot accounts" can not see imported sessions? 
![mk_1](https://github.com/user-attachments/assets/54918a21-d197-438b-8ce7-e80eae18a7d5)

should we import accounts again for each bot?
![image](https://github.com/user-attachments/assets/5a67a6cf-2412-430b-9ee0-9b4e96fa32c6)

PS. I've already activated my accounts (Enabled them) in "Manage accounts" section

**Comment by Fy0urM** [comment]
bot accounts are accounts added using tgInitData, so these accounts are individual for each individual bot. they are not related to accounts added via pyrogram.

**Reply by omidRMR** [reply]
thank you for your reply. So if I've imported accounts before, using sessions, I do not need to import anything in this step?

**Reply by Fy0urM** [reply]
Read this
https://github.com/masterking32/MasterCryptoFarmBot/wiki/How-to:-Add-New-Accounts

**Reply by omidRMR** [reply]
I read the Wiki page but I didn't find my answer.
- I've already imported accounts using pyrogram. Accounts are enabled in "Manage accounts".
![image](https://github.com/user-attachments/assets/45173da5-de86-494e-a0da-459de5a4a260)

-  The "bot accounts" for hamster kombat module doesn't show any account. Do I need import them again in "bot accounts"?
- 
![mk_1](https://github.com/user-attachments/assets/2b359e96-876b-4b4f-884c-e886c0ca1112)

**Comment by Fy0urM** [comment]
Accounts added via pyrogram are global. works for any module. accounts added in the settings of a particular module are for use in that module only and will not work for use in other modules. accounts within a module are added using tgWebAppInitData. accounts added via pyrogram should not appear in module accounts.

**Reply by omidRMR** [reply]
thank you for the clarification.  the problem is that I cannot see any log from hamsterbot even if when the terminal is showing the the module is running:
![image](https://github.com/user-attachments/assets/f2f79434-9520-45e8-b6f3-e50ea42f5a3a)

![image](https://github.com/user-attachments/assets/e8e47615-0b22-4d89-9f04-8bca2092573b)

**Comment by Fy0urM** [comment]
this is really weird, are you running MCF via start_* or py main.py?

**Reply by omidRMR** [reply]
i used conda to create the environment so i start program with "python main.py"

**Comment by Fy0urM** [comment, marked as answer]
make sure you are using the latest version of MCF, that's all I can tell you.

**Comment by Fy0urM** [comment]
I don't know what conda is or how it works. Since you have deviated from the recommended instructions, you will have to solve this problem yourself or find someone who knows conda better than you do

**Reply by omidRMR** [reply]
Thank your last suggestion worked! I removed the folder and did the entire installation process again. Now it's working! thank you!

PS: When I mention "conda," I am referring to Anaconda.

**Reply by omidRMR** [reply]
can we modify the bot and use it as another module? (I want to limit the upgrade process to a certain level of diamond per card)

**Reply by Fy0urM** [reply]
Still don't know what it is) consider running via start_* scripts. They automatically update the project.

**Reply by Fy0urM** [reply]
Of course, you can try it if you have enough knowledge to do so.
